examples/kilo.cpp

```diff
@@ -5,17 +5,12 @@
 #include <stdexcept>
```

Comment:
The file `kilo.cpp` is taken from elsewhere, I put it here to show that one can take a linux specific terminal application and easily transform it to use `cpp-terminal` to make it multiplatform. Most of the changes below are for the upstream code at https://github.com/snaptoken/kilo-src/blob/bc1039e7e2ac49ca6d0628ccd76c699b0736e1f4/kilo.c. I fine with the changes, they are relatively small.

Reply:
Nice to know that. I would have offered to try on merging it into upstream, but the new headers are part of std and therefore C++ exclusive.